js/main.js

```diff
@@ -866,10 +880,23 @@ function init() {
     autoPromoteElement.checked = gameData.autoPromoteEnabled;
     initStationName();
 
+    gameData.currentBattle = new Battle({name: 'The Destroyer', maxXp: 50, income: 5}, 5, 'battleProgressBar');
+    gameOverMessage.hidden = true;
+
     update();
     setInterval(update, 1000 / updateSpeed);
     setInterval(saveGameData, 3000);
     setInterval(setSkillWithLowestMaxXp, 1000);
 }
 
+Events.GameOver.subscribe(function (data) {
+    gameOverMessage.hidden = false;
+
+    if (data.bossDefeated) {
+        gameOverMessage.textContent = GameOverMessageWin;
+    } else {
+        gameOverMessage.textContent = GameOverMessageLose;
+    }
+});
+

```

Comment:
Ja nice.

Aber bitte einmal mit Zeile 883 in eine neue function schmeißen und die Function dann aus `init` herausaufrufen. kA, iwie `initBattle`  oder so